src/openloco/graphics/gfx.cpp

```diff
@@ -179,10 +179,82 @@ namespace openloco::gfx
      */
     uint16_t get_string_width(const char* buffer)
     {
-        registers regs;
-        regs.esi = (uintptr_t)buffer;
-        call(0x495685, regs);
-        return regs.cx;
+        uint16_t width = 0;
+        uint8_t* str = (uint8_t*)buffer;
+
+        while (*str != (uint8_t)0)
```

Comment:
Locomotion does not use Unicode, so all characters fit in one byte. However, we should be using `localisation::readCodePoint` to read one code point from a buffer.

Reply:
Ok, it is in the following commit.